requiam/commons.py

```diff
@@ -31,6 +34,59 @@ def figshare_stem(stem='', production=True):
     return stem_query
 
 
+def dict_load(config_file, vargs=None):
+    """
+    Purpose:
+      Read in a config INI file using configparser and return a dictionary
+      with sections and options
+
+    :param config_file: str. Full/relative path of configuration file
+    :param vargs: Dictionary containing command-line arguments
+    :return config_dict: dict of dict with hierarchy of sections follow by options
```

Comment:
*followed
(idk if I should be nit-picky about this stuff, will skip for now since its not code-related)

Reply:
You mean the whole Dictionary vs dict?

Reply:
No, there was a typo saying `follow by options` instead of `followed` but it's not a big deal.